back-end/sports_platform/sports_platform_api/tests/test_helper_functions.py

```diff
@@ -0,0 +1,6 @@
+from ..controllers.guest import Guest
+
+def create_mock_user(user_info):
+    guest= Guest(user_info['identifier'], user_info['password'])
+    guest.register(user_info)
+
```

Comment:
Can you return the user here also
```suggestion
return guest
```

Reply:
sure